extensions/rejected/WEBGL_shared_resources/extension.xml

```diff
@@ -370,5 +370,10 @@ interface WebGLTexture : WebGLSharedObject {
     <revision date="2014/07/15">
       <change>Added NoInterfaceObject extended attribute to extension interface and WebGLShareGroup.</change>
     </revision>
+    <revision date="2018/05/02">
+      <change>Rejected after discussion on public_webgl and no strong objections. At this point in
+      the WebGL API's development it is not profitable to invest the time to implement this
+      extension.</change>
```

Comment:
It may be worth mentioning the alternative(s) here.

Reply:
Thanks for the suggestion. Described the alternatives.